.github/workflows/main.yml

```diff
@@ -0,0 +1,218 @@
+---
+name: CI
+
+"on":
+  push:
+    branches:
+      - "main"
+    tags:
+      - "v*.*.*"
+  pull_request:
+    branches:
+      - "main"
+
+permissions:
+  contents: read
+
+env:
+  FPM_VERSION: 1.15.1
```

Comment:
Remove all references to Ruby and FPM, since they're used for Debian packaging which this project does not support.